src/common/paths/ping.yml

```diff
@@ -0,0 +1,30 @@
+get:
+  operationId: PostPing
```

Comment:
Can you make these `operationId`s unique?

Reply:
Fixed